docs/install/windows.md

```diff
@@ -1,37 +1,392 @@
+
+
+### Step 14: Using and modify NEURON file... 
+You can replace any new `.dll` file at `C:\nrn\bin` and the NEURON will prevent the new feature implemented. 
+ 
+Example: 
+
+#### Step 14.A: Checking the `nrngui` 
+Run `nrngui` (inside the folder `C:\Users\User\Desktop\NEURON 9.0.0 AMD64` double click on `nrngui`)
+
+It will show something like:
+
+```
+NEURON -- VERSION X.X.X 2024-07-09
+
+Duke, Yale, and the BlueBrain Project -- Copyright 1984-2022
+See http://neuron.yale.edu/neuron/credits
+
+oc>
+```
+
+Close this neuron window.
+
+#### Step 14.B: Making some changes at `init.cpp`
+
+Open the `init.cpp` file (at `C:\Users\User\Neuron\nrn\src\nrnoc\init.cpp`  or at `C:\msys64\home\User\Neuron\nrn\src\nrnoc\init.cpp`) look for `hoc_last_init(void)` and the follow code inside it:
+
+```C
+    if (nrnmpi_myid < 1)
+        if (nrn_nobanner_ == 0) {
+            Fprintf(stderr, "%s\n", nrn_version(1));
+            Fprintf(stderr, "%s\n", banner);
+            IGNORE(fflush(stderr));
+        }
 ```
-in order to:
 
-* install all downloaded Python versions and for each:
-  * fix MSVCC version
-  * fix MSVC runtime library
-  * install numpy 
-* install `nsis` and required plugin
-* install MPI
-* install MSYS2 (via `Chocolatey`) and then MinGW toolchain and required build pacakages
+
+modifying this introduction the print line `printf("\n Now It is my NEURON version!!!! \n\n");` for example: 
+
+```C
+     if (nrnmpi_myid < 1)
+        if (nrn_nobanner_ == 0) {
+            Fprintf(stderr, "%s\n", nrn_version(1));
+            printf("\nNow It is my NEURON version!!!! \n\n");
+            Fprintf(stderr, "%s\n", banner);
+            IGNORE(fflush(stderr));
+        }
+```
+and save.
+
+#### Step 14.C: Compiling the new code.
+
+Compile this new code. We can compile using Step 15. (Do it and come back here) 
+
+#### Step 14.D: Replacing the `libnrniv.dll` file
+
+It will end up creating a new `libnrniv.dll` (and `libnrnmpi_msmpi.dll` and a new `libnrnpython3.XX.dll` and new `nrniv`, but just the first one matter for THIS example) at `nrn\build\bin` (at `C:\Users\User\Neuron\nrn\build\bin` or at `C:\msys64\home\User\Neuron\nrn\build\bin` if you did the installation process here). 
+
+We can replace the old file `libnrniv.dll` at `C:\nrn\bin` for this new one and voila! We get our change implemented!! Let`s see
+
+#### Step 14.E: Checking the new `nrngui` 
+
+Run `nrngui` (inside the folder `C:\Users\User\Desktop\NEURON 9.0.0 AMD64` double click in `nrngui`)
+
+```
+NEURON -- VERSION X.X.X 2024-07-11
+
+Now it is my NEURON version!!!!
+
+Duke, Yale, and the BlueBrain Project -- Copyright 1984-2022
+See http://neuron.yale.edu/neuron/credits
+
+oc>
+```
+
+------------------------------------------------------------ 
+
+### Step 15: Compile new Neuron code
+
+
+PS: Useful commands:  
+
+```
+cd /c/Users/User/Neuron/nrn/build
+``` 
+to reach files outside of `C:\msys64\`.
+
+```
+Echo $PATH
```

Comment:
this works because windows is case insensitive, but usually `echo` and `which` are written in lowercase because that's what they have to be in linux.